lib/services/local/events.go

```diff
@@ -383,6 +387,9 @@ func (p *certAuthorityParser) parse(event backend.Event) (types.Resource, error)
 		if err != nil {
 			return nil, trace.Wrap(err)
 		}
+		if !p.filter.Match(ca) {
```

Comment:
I don't think that this helps, the `UnmarshalCertAuthority` immediately above will still error out if the auth server sees an event for a CA type that it doesn't know about.

Reply:
It's so that filtering still works with cache disabled - not to help auth

Reply:
But the auth will never run a CA filter, right?

Reply:
auth's cache won't be configured with a CA filter.

When gRPC `WatchEvents` is called, if cache is enabled it will attach to auth cache's fanout.
If cache is disabled, then it creates a new backend watcher.

I'm guessing that disabling auth cache pretty much never happens, but filtering should still work if it is disabled.